miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/Type.kt

```diff
@@ -51,5 +52,6 @@ enum class NativeEventType(val value: String) {
     MINIAPP_ON_PAUSE("miniapppause"),
     MINIAPP_ON_RESUME("miniappresume"),
     MINIAPP_SECURE_STORAGE_READY("miniappsecurestorageready"),
-    MINIAPP_SECURE_STORAGE_LOAD_ERROR("miniappsecurestorageloaderror")
+    MINIAPP_SECURE_STORAGE_LOAD_ERROR("miniappsecurestorageloaderror"),
+    RECEIVE_UNIVERSAL_JSON_INFO("miniappreceivejsoninfo")
```

Comment:
```suggestion
    RECEIVE_JSON_INFO("miniappreceivejsoninfo")
```

Reply:
For maintainability reasons, I would suggest keeping it that way. However, I'm okay with the change either.